nepalingo-web/src/App.tsx

```diff
@@ -20,7 +20,7 @@ const App: React.FC = () => {
     <Router>
       <Routes>
         <Route path="/login" element={<User_auth />} />
-        <Route path="/learn" element={<FlashcardPage />} />
+        <Route path="/flashcard" element={<FlashcardPage />} />
```

Comment:
This needs to check for user similar to like 25 and redirect to login if user is not logged in